app/src/main/java/com/example/poste/http/EditPostRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.example.poste.http;
+
+public class EditPostRequest {
```

Comment:
Are we doing anything with tags?

Reply:
Not at the moment. It's a future feature, I will comment//hide it in the layout for now